ci/integration_tests/Cargo.toml

```diff
@@ -1,16 +1,24 @@
 [package]
-name = "integration_tests"
+name = "integration_tests" # TODO: Rename this to `integration-tests` to be like everything else
 version = "0.1.0"
 edition = "2021"
 
+# We conditionally enable the optimize feature
+# because compiling and building cw-optimizoor takes a lot of time.
+# This way we can save some minutes for non integration test builds
+# by defaulting to not building the dependencies.
```

Comment:
You can see [here](https://github.com/DA0-DA0/dao-contracts/runs/7910928593?check_suite_focus=true) that before when I simply enabled the optimize feature globally, the lint CI job went from ~6 min -> ~11 min runtime.